test/getters.test.js

```diff
@@ -0,0 +1,300 @@
+import { jDataView } from "../src/jdataview";
+import {
+	compareBytes,
+	compareWithNaN,
+	getPrefilledJDataView,
+	b,
+	chr,
+	bufferToHex,
+} from "./test-helpers";
+import { describe, test, assert } from "vitest";
+
+describe("Getters", function () {
+	// getter = value || {value, check?, view?, args?}
+	function testGetters(type, getters) {
+		const view = getPrefilledJDataView();
+		test(type, function () {
+			getters.forEach(function (getter) {
+				if (typeof getter !== "object") {
+					getter = { value: getter };
+				}
+
+				const args = getter.args || [];
+				const contextView = getter.view || view;
+				const check = getter.check || assert.equal;
+				const value = getter.value;
+				const offset = contextView.tell();
+				const realValue = contextView["get" + type].apply(
+					contextView,
+					args
+				);
+
+				check(
+					realValue,
+					value,
+					"get" +

```

Comment:
Wow this pre-ES6 code is something when formatted. Nit, but since you converted the rest to ES6, can this be turned into template literal for readability please?